packages/files-ui/src/Components/Modules/FileBrowsers/views/FileSystemItem/FileSystemGridItem.tsx

```diff
@@ -180,7 +180,7 @@ const FileSystemGridItem = React.forwardRef(
       }
 
       return {
-        fileName : name.replace(extension, ""),
+        fileName: name.slice(0, name.length - extension.length - 1),
```

Comment:
Whoops, was not necessary for that `- 1` to be there, since the extension already includes the `.`
```suggestion
        fileName: name.slice(0, name.length - extension.length),
```